### PR TITLE
chore: move up the examples folder's `.gitignore` and `.gitattributes` files to the root directory

### PR DESCRIPTION
**Pull request by poppoerika**
Per @malandis's feedback on [this pr](https://github.com/momentohq/client-sdk-java/pull/145), so that the project can pick up these files.

### CODE REVIEW

**Comment by poppoerika** [comment]
Also moved dependabot.

**Comment by cprice404** [comment]
sorry one more thing!  There is a ci job in the examples repo that build/runs the examples when pull requests are filed.  we need that, so that when dependabot files PRs, they will be tested.

**Comment by cprice404** [comment]
LGTM except the build is failing.  I think the build is failing because we are missing the gradle jar file in the `examples/gradle/wrapper` dir.

**Comment by poppoerika** [comment]
Yeah, I was able to build it locally and when I googled the error I got the same answer as your suggestion!
I'll add `gradle-wrapper.jar` to `examples/gradle/wrapper`

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gradle" # See documentation for possible values
+    directory: "/examples" # Location of package manifests
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "momento-sdk"
```
Comment (malandis) [inline .github/dependabot.yml:13]: Newline at end of file needed ^

**Review by malandis — COMMENTED** [review]
Thanks. Just a small change noted below

**Review by cprice404 — APPROVED** [review]
:ty:

**Pull request merged**